composer.json

```diff
@@ -22,6 +22,7 @@
     "easyframework/collections": "dev-master"
   },
   "require-dev": {
+    "php-amqplib/php-amqplib": "2.6.*",
```

Comment:
The dev requirement will be installed unless you used *--no-dev* but just for the scope of this pacakge.
They won't be installed if you run composer install or update in another package that uses this one. http://stackoverflow.com/a/19120691